IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailTypeCheckRail/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailTypeCheckRail" UniqueId="22a83034-ec44-415a-b5fd-1509d33d9a00" ApplicableType="IfcRail/CHECKRAIL">
```

Comment:
Just FYI, bSI was complaining if the `PropertySetType` was not set. I know, it is not mandatory, but TKrijnen posted an issue that these are not set.

Reply:
OK, added: https://github.com/bSI-InfraRoom/IFC-Specification/pull/163/commits/0f6a2672e05cf6089ccc9c1aa1e68ff3a7b4a16e